### PR TITLE
[Workers] Cron Triggers revision

### PR DESCRIPTION
**Pull request by deadlypants1973**
PCX-6579

### CODE REVIEW

File: content/workers/examples/cron-trigger.md [hunk 1]
```diff
@@ -60,9 +50,21 @@ name = "worker"
...
+## Test Cron Triggers using Wrangler
+
+The recommended way of testing Cron Triggers is using Wrangler.
+
+Cron Triggers can be tested using Wrangler by passing in the `--test-scheduled` flag to [`wrangler dev`](/workers/wrangler/commands/#dev). This will expose a `/__scheduled` route which can be used to test using a HTTP request. To simulate different cron patterns, a `cron` query parameter can be passed in.
+
+```sh
+$ wrangler dev --test-scheduled
+
+$ curl "http://localhost:8787/__scheduled?cron=*+*+*+*+*"
```
Comment (maxwellpeterson) [inline content/workers/examples/cron-trigger.md]:
```suggestion
$ curl "http://localhost:8787/__scheduled?cron=0+*+*+*+*"
```
Update cron expression in URL to match cron expression defined in `wrangler.toml`

File: content/workers/configuration/cron-triggers/index.md [hunk 2]
```diff
@@ -8,25 +8,56 @@ layout: single
...
+
+Refer to the example below for a sample `wrangler.toml` Cron Triggers configuration:
+
+```toml
+[triggers]
+crons = [ "*/3 * * * *", "0 15 1 * *", "59 23 LW * *" ]
+# Schedule crons triggers:
+# - At every 3rd minute
+# - At 3PM on first day of the month
+# - At 11:59PM on the last weekday of the month
```
Comment (maxwellpeterson) [inline content/workers/configuration/cron-triggers/index.md]:
```suggestion
# Schedule cron triggers:
# - At every 3rd minute
# - At 3PM on first day of the month
# - At 11:59PM on the last weekday of the month
crons = [ "*/3 * * * *", "0 15 1 * *", "59 23 LW * *" ]
```
We should put this comment above the cron definition array

File: content/workers/configuration/cron-triggers/index.md [hunk 3]
```diff
@@ -8,25 +8,56 @@ layout: single
 
 ## Background
 
-Cron Triggers allow users to map a cron expression to a Worker script using a [ScheduledEvent](/workers/runtime-apis/scheduled-event/) listener that enables Workers to be executed on a schedule. Cron Triggers are ideal for running periodic jobs for maintenance or calling third-party APIs to collect up-to-date data. Workers scheduled by Cron Triggers will run on underutilized machines to make the best use of Cloudflare's capacity and route traffic efficiently.
+Cron Triggers allow users to map a cron expression to a Worker using a [`ScheduledEvent`](/workers/runtime-apis/scheduled-event/) listener that enables Workers to be executed on a schedule.
+
+Cron Triggers are ideal for running periodic jobs for maintenance or calling third-party APIs to collect up-to-date data. Workers scheduled by Cron Triggers will run on underutilized machines to make the best use of Cloudflare's capacity and route traffic efficiently.
```
Comment (Skye-31) [inline content/workers/configuration/cron-triggers/index.md]: Small nit 🙂
```suggestion
Cron Triggers are ideal for running periodic jobs, such as for maintenance or calling third-party APIs to collect up-to-date data. Workers scheduled by Cron Triggers will run on underutilized machines to make the best use of Cloudflare's capacity and route traffic efficiently.
```

File: content/workers/configuration/cron-triggers/index.md [hunk 4]
```diff
@@ -8,25 +8,56 @@ layout: single
...
 
 ## Add a Cron Trigger
 
-Add Cron Triggers to Workers with the Cloudflare API, or in the dashboard. To add Cron Triggers in the Cloudflare dashboard:
+### 1. Define a scheduled event listener
 
-1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com) and select your account.
-2. In Account Home, select **Workers & Pages**.
-3. In **Overview**, select your Worker > **Triggers** > **Cron Triggers**.
+To respond to a Cron Trigger, you must add a [`"scheduled"` event](/workers/runtime-apis/scheduled-event/) listener, or export a default `scheduled` handler to your Workers code.
```
Comment (marciocloudflare) [inline content/workers/configuration/cron-triggers/index.md:21]: Is it `ScheduledEvent` or `"scheduled"` event? Lines 11 and 21 disagree on this.

File: content/workers/configuration/cron-triggers/index.md [hunk 5]
```diff
@@ -8,25 +8,56 @@ layout: single
...
+* [Setting Cron Triggers](/workers/examples/cron-trigger/)
+* [Multiple Cron Triggers](/workers/examples/multiple-cron-triggers/)
 
-To respond to a Cron Trigger, you must add a [`"scheduled"` event](/workers/runtime-apis/scheduled-event/) listener, or export a default `scheduled` handler to the Workers script.
+### 2. Update configuration
 
-{{</Aside>}}
+After you have updated your Worker code to include a `"scheduled"` event, you must update your Worker project configuration.
+
+#### via `wrangler.toml`
```
Comment (marciocloudflare) [inline content/workers/configuration/cron-triggers/index.md]:
```suggestion
#### Via `wrangler.toml`
```

File: content/workers/configuration/cron-triggers/index.md [hunk 6]
```diff
@@ -8,25 +8,56 @@ layout: single
...
+```
+
+You also can set a different Cron Trigger for each [environment](/workers/wrangler/environments/) in your `wrangler.toml`. You need to put the `[triggers]` table under your chosen environment. For example:
+
+```toml
+[env.dev.triggers]
+crons = ["0 * * * *"]
+```
+
+#### via the dashboard
```
Comment (marciocloudflare) [inline content/workers/configuration/cron-triggers/index.md]:
```suggestion
#### Via the dashboard
```

**Review by marciocloudflare — APPROVED** [review]

**Pull request merged**